permissions/plugin-prisma-cloud-iac-scan.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "prisma-cloud-iac-scan"
+github: "jenkinsci/prisma-cloud-iac-scan-plugin"
+paths:
+- "org/jenkins-ci/plugins/prisma-cloud-iac-scan"
```

Comment:
The start of the path is your group id and this doesn’t match what your plugin has

Reply:
@timja We have updated the path, Could you please check?

Reply:
@timja  We updated the path in yml and pom correctly. Could you please review it?

Reply:
Please update the yml file and put the pom.xml value back to what it was before.

Reply:
We just updated. Please check, thanks.